Tevinthuku/django-teammates
# #164743694 Create New Teammate

### What does this PR do?
This PR adds the functionality necessary to create a new teammate

### Tasks to be completed?

- [x] - Create a new teammate via the post method

### How should this be manually tested?

- clone this repo and check-out to the `ft/164743694-user-create-teammate` branch
- start the app
- go to `http://localhost:8000/team/all/` and create a new teammate.

### PT stories relevant to this PR?
[#164743694](https://www.pivotaltracker.com/n/projects/2320376/stories/164743694)
